builtin/files/userdata/cloud-config-controller

```diff
@@ -1449,6 +1449,13 @@ write_files:
                 #  value: "true"
                 #- name: TYPHA_PROMETHEUSMETRICSPORT
                 #  value: "9093"
+              resources:
```

Comment:
What you could do here is to put your default values into the function NewDefaultCluster() within pkg/api/cluster.go?  Then you don't need to do the conditional in the template... could you do that?